doc/style.css

```diff
@@ -347,6 +347,22 @@ div.appendix div.example {
     margin-top: 1.5em;
 }
 
+div.book div.example details,
+div.appendix div.example details {
```

Comment:
Your CSS assumes a very specific HTML structure. If the HTML changes, such as adding another parent div this will break.
Id suggest '.example details' 
which targets all details with a parent of class .example
this would allow other parents as well. making it more resistent to potential future changes.

Reply:
I prefer to keep the specific CSS here - it's only used with the output of nrd, and we control the HTML structure there. It's expected that changes to nrd will be at least rendered so we can glance over the output, and if the CSS needs to be adapted (such as in this case), then we'll just change the CSS as well.

To explain why I'm taking this stance: this CSS file has a few rules that conflict on certain elements, and we rely on the more specific rule overriding the value of the less specific one.
As an exercise, I tried removing all the `div.book` and `div.appendix` specifiers from all the other rules as well, and some things got messed up.
I prefer keeping the specific rules because that's how the rest of the CSS is laid out, so it avoids subtle things from breaking if someone decides to change some other less-specific rule.